src/pages/ProductDetail/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import styles from './Comment.module.css';
+import { useParams } from 'react-router-dom';
+import { postProductComment, getProductComments } from '../../../../api/itemApi';
+import CommentItem from '../../components/CommentItem/CommentItem';
+
+function Comment() {
+  const [value, setValue] = useState('');
+  const [isValid, setIsValid] = useState(false);
+  const { productId } = useParams();
+  const [itemList, setItemList] = useState([]);
+  const LIMIT = 6;
+  const [cursor, setCursor] = useState(null);
+  
+  const handleLoad = useCallback(async () => {
+      const comments = await getProductComments(productId, { limit: LIMIT, cursor:0 });
+      setItemList(comments.list); 
+      console.log(comments.list)
+  }, [productId, cursor]);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const formData = new FormData();
+    formData.append('comment', value);
+    await postProductComment(productId, formData);
+    setValue('');
```

Comment:
P3:
form에 다른 값이 있는 것도 아니라서 formData를 쓰기보다 그냥 value로 접근하는게 더 좋을 것 같아요.

```suggestion
    await postProductComment(productId, value);
    setValue('');
```